commands/provider/service_deployer.go

```diff
@@ -57,7 +61,12 @@ func (p *ServiceProvider) ServiceDeploy(path string, env map[string]string, stat
 	go readDeployReply(stream, deployment, statuses)
 
 	if govalidator.IsURL(path) {
```

Comment:
The following string is not recognized as a URL:
`mesg.marketplace:<sid>:<hash>`